cmd/edgex-secrets-config/README.md

```diff
@@ -0,0 +1,193 @@
+% edgex-secrets-config-proxy(1) User Manuals edgex-secrets-config-proxy(1)
+
+# NAME
+
+edgex-secrets-config-proxy – Configure EdgeX proxy service
+
+# SYNOPSIS
+
+**edgex-secrets-config proxy** SUBCOMMAND [OPTIONS]
+
+# DESCRIPTION
+
+Configures the EdgeX proxy service.
+
+This command is used to configure the TLS certificate for external connections, create authentication tokens for inbound proxy access, and other related utility functions.
+
+Proxy configuration commands (listed below) require access to the secret store master key in order to generate temporary secret store access credentials.
+
+# OPTIONS
+
+  * **--confdir** _/path/to/directory/with/configuration.toml_ (optional)
+
+    Points to directory containing a configuration.toml file.
+
+# SUBCOMMANDS
+
+  * **tls**
+
+    Configure inbound TLS certificate. This command will provision the TLS secrets into the secret store and re-deploy them to Kong. Requires additional arguments:
+
+    * **--in** _/path/to/certchain.crt_ (required)
+
+      Path to TLS leaf certificate (PEM-encoded x.509) (the file extension is arbitrary).
+      If intermediate certificates are required to chain to a certificate authority,
+      these should also be included.
+      The root certificate authority should not be included.
+
+    * **--inkey** _/path/to/private.key_ (required)
+
+      Path to TLS private key (PEM-encoded).
+
+  * **adduser**
+
+    Create a proxy user using specified token type. Requires additional arguments:
+
+    * **--token-type** jwt | oauth2 (required)
+    
+      Create user using either the JWT or OAuth2 authentication plugin.
+      This value must match the configured authentication plugin.
+
+    * **--user** _username_ (required)
+
+      Username of the user to add.
+
+    * **--group** _group_ (optional)
+
+      Group to which the user belongs, defaults to &quot;admin&quot;.
+      This should be the group associated with the route ACL.
+      This value can be changed by editing the `[KongACL]` section of security-proxy-setup's configuration.toml.
```

Comment:
Is `security-proxy-setup's` correct? or should it be `edgex-secrets-config's`?

Reply:
Good catch.  I want them to share the same config file, maybe I should note this.